### PR TITLE
DRAFT: try to reproduce #620

### PR DESCRIPTION
**Pull request by thiagowfx**

### CODE REVIEW

**Comment by thiagowfx** [comment]
The test is passing, but I was purposely trying to make it fail in order to reproduce the bug. I could not.

File: tests/test_browsing_context.py [hunk 1]
```diff
@@ -732,6 +732,29 @@ async def test_browsingContext_navigateSameDocumentNavigation_waitComplete_navig
...
 
+@pytest.mark.asyncio
+async def test_browsingContext_logAfterNavigation(websocket, context_id, html):
+    await subscribe(websocket, "log.entryAdded")
+
+    await send_JSON_command(
+        websocket, {
+            "method": "browsingContext.navigate",
+            "params": {
+                "url": html("<script>console.log('mylog');</script>"),
```
Comment (sadym-chromium) [inline tests/test_browsing_context.py:743]: try OOPiF sending log. I believe the root cause is there

Comment (sadym-chromium) [reply tests/test_browsing_context.py:743]: Eg you can use Mathias' demo (as long as our e2e is not local anyways)

**Comment by thiagowfx** [comment]
During the stand-up yesterday, we discussed using `another_context_id` to trigger the failure. The test still passes though.

**Pull request closed without merging**